build/pandoc/defaults/html.yaml

```
@@ -23,7 +23,7 @@ include-after-body:
 - build/plugins/analytics.html
 - build/plugins/hypothesis.html
 - build/plugins/mathjax.html
-#- build/plugins/scite.html
+# - build/plugins/scite.html
```

```suggestion
#- build/plugins/scite.html
```
Removing the space here so that the whitespace matches the other commented out plugins.